src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -3,43 +3,35 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Device;
 using System.Device.Gpio;
+using System.Device.I2c;
 using System.Diagnostics;
 using Iot.Units;
 
 namespace Iot.Device.DHTxx
 {
-    public class DHTSensor : IDisposable
+    /// <summary>
+    /// Temperature and Humidity Sensor DHTxx
+    /// </summary>
+    public class DhtSensor : IDisposable
     {
-        private const int MAX_TIME = 85;
-        private const uint MAX_WAIT = 255;
-        private byte[] _dht11Val = new byte[5];
-        private int _pin;
-        private DhtType _dhtType;
+        /// <summary>
+        /// DHT12 Default I2C Address
+        /// </summary>
+        public const byte Dht12DefaultI2cAddress = 0x5C;
```

Comment:
should this be just `DefaultI2cAddress` or are other Dhts different? If different I'd still rename to `DefaultI2cAddressDh12` so that they are sorted in such a way that it is easier to find